tests/test_transforms/test_math/test_differencing_transform.py

```diff
@@ -101,17 +109,18 @@ def check_transform(
         assert np.all(series_init == series_transformed)
 
 
-def check_inverse_transform_not_inplace(transform: GeneralDifferencingTransform, df: pd.DataFrame):
+def check_inverse_transform_not_inplace(transform: GeneralDifferencingTransform, ts: TSDataset):
     """Check that differencing transform does nothing during inverse_transform in non-inplace mode."""
-    transformed_df = transform.fit_transform(df)
-    inverse_transformed_df = transform.inverse_transform(transformed_df)
+    transformed_df = transform.fit_transform(ts).to_pandas()
+    inverse_transformed_df = transform.inverse_transform(ts).to_pandas()
     assert transformed_df.equals(inverse_transformed_df)
 
 
-def check_inverse_transform_inplace_train(transform: GeneralDifferencingTransform, df: pd.DataFrame):
+def check_inverse_transform_inplace_train(transform: GeneralDifferencingTransform, ts: TSDataset):
     """Check that differencing transform correctly makes inverse_transform on train data in inplace mode."""
-    transformed_df = transform.fit_transform(df)
-    inverse_transformed_df = transform.inverse_transform(transformed_df)
+    df = ts.to_pandas()
+    transform.fit_transform(ts)
+    inverse_transformed_df = transform.inverse_transform(ts).to_pandas()
     assert inverse_transformed_df.equals(df)
```

Comment:
Let's replace it with `pd.testing.assert_frame_equal`, it gives more information about errors.